pkg/filter/filter.go

```diff
@@ -85,15 +100,17 @@ func (f VulnerabilityFilter) FilterVulnerabilities(ctx context.Context, asset do
 				Method:  logs.VulnID,
 				VulnID:  vuln.ID,
 				AssetID: asset.ID,
+				Status:  vuln.Status,
 			})
 			stater.Count("event.nexposevulnerability.filter.accepted", 1)
 		default:
 			logger.Info(logs.VulnerabilityFiltered{
 				Action:  logs.VulnDiscarded,
 				VulnID:  vuln.ID,
 				AssetID: asset.ID,
+				Status:  vuln.Status,
 			})
-			stater.Count("event.nexposevulnerability.filter.discarded", 1)
+			stater.Count("event.nexposevulnerability.filter.discarded", 1, fmt.Sprintf("reason:%s", "default"))
```

Comment:
Suggestion: I'd change `"default"` to `"unknown"` and make it a constant.

Reply:
fixed